src/pudl/output/ferc1.py

```diff
@@ -3053,36 +3049,69 @@ def out_ferc1__yearly_rate_base(
             disaggregate_null_or_total_tag,
             tag_col="tags_aggregatable_utility_type",
         )
+        # bc we just disaggreated the utility type tag, there are some
+        # totals in the utilty_type column. even though these two columns
+        # do not need to stay consistent, it feels cleaner to keep
+        # these consistent for these dissagregated records
+        .assign(
+            utility_type=lambda x: np.where(
+                x.is_disaggregated_tags_aggregatable_utility_type,
+                x.tags_aggregatable_utility_type,
+                x.utility_type,
+            )
+        )
         .pipe(
             disaggregate_null_or_total_tag,
             tag_col="tags_in_rate_base",
         )
     )
 
     in_rate_base_df = rate_base_df[rate_base_df.tags_in_rate_base == "yes"]
-    return in_rate_base_df.dropna(subset=["ending_balance"])
-
-
-@asset_check(
-    asset="out_ferc1__yearly_rate_base",
-    additional_ins={"tags_df": AssetIn("_out_ferc1__detailed_tags")},
-    blocking=True,
-)
-def check_tag_propagation(out_ferc1__yearly_rate_base, tags_df: pd.DataFrame):
-    """Check propagated tags compared to manually compiled tags."""
     for tag in ["in_rate_base", "aggregatable_utility_type"]:
         # note: we need the `tags_in_rate_base` column for these checks
         check_tag_propagation_compared_to_compiled_tags(
-            out_ferc1__yearly_rate_base, tag, tags_df
+            in_rate_base_df, tag, _out_ferc1__detailed_tags
         )
-    return AssetCheckResult(passed=True)
+        check_for_correction_xbrl_factoids_with_tag(in_rate_base_df, tag)
+    return in_rate_base_df.dropna(subset=["ending_balance"])
```

Comment:
Curious what goes into the decision to make something and asset check vs. a regular, defensive in-line check?

Reply:
this is documented in the issue